contrib/endpoints/repositories.bzl

```diff
@@ -193,143 +193,36 @@ def grpc_repositories(bind=True):
             actual = "@grpc_git//:grpc++_codegen_proto",
         )
 
-def googleapis_repositories(protobuf_repo="@protobuf_git//", bind=True):
```

Comment:
this is here indicating that your change is base is behind latest of rate_limiting branch.  Could you rebase your change?